packages/shared/lib/core/account/interfaces/account-metadata.interface.ts

```diff
@@ -4,4 +4,5 @@ export interface IAccountMetadata {
     color: string
     hidden: boolean
     shouldRevote: boolean
+    removedProposalsIds?: string[]
```

Comment:
Slight typo in the name, it should be `removedProposalIds` rather than `removedProposalsIds`